gateway-export-plugin/src/test/java/com/ca/apim/gateway/cagatewayexport/tasks/explode/linker/EncassLinkerTest.java

```diff
@@ -6,22 +6,47 @@
 
 package com.ca.apim.gateway.cagatewayexport.tasks.explode.linker;
 
-import com.ca.apim.gateway.cagatewayconfig.beans.Bundle;
-import com.ca.apim.gateway.cagatewayconfig.beans.Encass;
-import com.ca.apim.gateway.cagatewayconfig.beans.Folder;
-import com.ca.apim.gateway.cagatewayconfig.beans.FolderTree;
+import com.ca.apim.gateway.cagatewayconfig.beans.*;
+import com.ca.apim.gateway.cagatewayconfig.util.xml.DocumentParseException;
+import com.ca.apim.gateway.cagatewayconfig.util.xml.DocumentTools;
+import com.ca.apim.gateway.cagatewayconfig.util.xml.DocumentUtils;
 import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.Test;
 
+import java.util.HashMap;
+
+import static com.ca.apim.gateway.cagatewayconfig.util.policy.PolicyXMLElements.API_PORTAL_ENCASS_INTEGRATION;
+import static com.ca.apim.gateway.cagatewayconfig.util.properties.PropertyConstants.*;
+import static com.ca.apim.gateway.cagatewayconfig.util.xml.DocumentUtils.getSingleElement;
 import static com.ca.apim.gateway.cagatewayexport.util.TestUtils.*;
 import static org.apache.commons.lang3.StringUtils.EMPTY;
-import static org.junit.jupiter.api.Assertions.assertEquals;
-import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.*;
 
 class EncassLinkerTest {
     private EncassLinker encassLinker;
     private Encass myEncass;
     private Bundle bundle;
+    private static final String ENCASS_POLICY_WITH_PORTAL_INTEGRATION = "<?xml version=\"1.0\" encoding=\"UTF-8\"?>\n" +
+            "<wsp:Policy xmlns:L7p=\"http://www.layer7tech.com/ws/policy\" xmlns:wsp=\"http://schemas.xmlsoap.org/ws/2002/12/policy\">\n" +
+            "    <wsp:All wsp:Usage=\"Required\">\n" +
+            "        <L7p:ApiPortalEncassIntegration/>\n" +
+            "        <L7p:CommentAssertion>\n" +
+            "            <L7p:Comment stringValue=\"Policy Fragment: includedPolicy\"/>\n" +
+            "        </L7p:CommentAssertion>\n" +
+            "    </wsp:All>\n" +
+            "</wsp:Policy>";
+
+    private static final String ENCASS_POLICY_WITH_PORTAL_INTEGRATION_DISABLED = "<?xml version=\"1.0\" encoding=\"UTF-8\"?>\n" +
+            "<wsp:Policy xmlns:L7p=\"http://www.layer7tech.com/ws/policy\" xmlns:wsp=\"http://schemas.xmlsoap.org/ws/2002/12/policy\">\n" +
+            "    <wsp:All wsp:Usage=\"Required\">\n" +
+            "        <L7p:ApiPortalEncassIntegration>\n" +
+            "            <L7p:Enabled booleanValue=\"false\"/>\n" +
```

Comment:
when booleanValue is true, do we consider that as Enabled case

Reply:
@uppoju , it will never be true, if this assertion is present and enabled in the policy, then it generates Element as <L7p:ApiPortalEncassIntegration/>
if assertion is present and disabled in policy, it generates Element as 
            <L7p:ApiPortalEncassIntegration>
                    <L7p:Enabled booleanValue="false"/>
            </L7p:ApiPortalEncassIntegration>

Reply:
OK